### PR TITLE
Prepare for repo archival

### PR DESCRIPTION
**Pull request by a1exsh**
Update documentation pages before archiving the repository.

### CODE REVIEW

**Comment by a1exsh** [comment]
:+1:

**Comment by adyach** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## [Nakadi Event Broker](https://zalando.github.io/nakadi/)
 
+**Nakadi is no longer developed in the open — please use Zalando's internal repository for contributing.**
```
[the remark this answers is not in the document]
Comment (leviferreira) [reply README.md:3]: Let's not add any references to internal repositories.

[the remark this answers is not in the document]
Comment (leviferreira) [reply README.md:3]:
```suggestion
**Nakadi is no longer developed in the open as we've not seen usage outside of Zalando for many years. **
If you are a Zalando employee, please use Zalando's internal repository for contributing.
```

[the remark this answers is not in the document]
Comment (ePaul) [reply README.md:3]: > Let's not add any references to internal repositories.

What is the harm? It makes it easier for Zalando people to find the right place, and external people will know not to click it (or if they click it, they won't be able to login anyways).

**Comment by a1exsh** [comment]
:+1:

**Comment by samizzy** [comment]
:+1:

**Pull request merged**